src/connectors/queue/likecoin.ts

```diff
@@ -94,6 +113,32 @@ class LikeCoinQueue extends BaseQueue {
       done(e)
     }
   }
+
+  private handleGetCivicLiker: Queue.ProcessCallbackFunction<unknown> = async (
+    job,
+    done
+  ) => {
+    try {
+      const { likerId } = job.data as GetCivicLikerData
+
+      const isCivicLiker = await this.userService.likecoin.isCivicLiker({
+        likerId,
+      })
+
+      // update cache
+      const cacheService = new CacheService(CACHE_PREFIX.CIVIC_LIKER)
+      cacheService.storeObject({
+        keys: { id: likerId },
+        data: isCivicLiker,
+        expire: CACHE_TTL.LONG,
+      })
+
+      job.progress(100)
+      done(null, { likerId, isCivicLiker })
+    } catch (e) {
+      done(e)
+    }
```

Comment:
the `try -catch` may not be necessary, I see the bull supports Promise and throw err,

the throw exception is taken care of automatically